wiki/release.md

```diff
@@ -0,0 +1,30 @@
+# Releases
+
+## Release workflow
+
+the flow should be goes as following
+
+1. Each release is developed against branch at the beginning and this is branched from development `i.e development-2.0`
+2. All `pull requests` should be to this branch no commits goes to `development` directly
```

Comment:
All pull requests should be made to this release branch; no commits should be made directly to the development branch.